utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -47,21 +45,22 @@ class HtmlParser private constructor(
 
     val imageGetter = urlImageParserFactory.create(htmlContentTextView, entityType, entityId, imageCenterAlign)
 
-    val htmlSpannable = HtmlCompat.fromHtml(htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
+    val formattedHtml = htmlContent
```

Comment:
This is a nice simplification of parsing & handling lists!

Reply:
Thanks.